benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py

```diff
@@ -44,7 +44,8 @@ def get_last_elasticsearch_versions(self, last_es_fetch_days=LAST_ES_FETCH_DAYS)
             data = self.elasticsearch.get_elasticsearch_index_by_id(index='ci-status', id=id)
             for version, value in data['_source'].items():
                 if version in display_versions and value not in new_versions.values() and len(value) < self.MAX_VERSION_LEN:
-                    new_versions[value.replace(".", "").replace("-", "")] = value
+                    # Display only numbers in the Grafana panel, removing characters
+                    new_versions[value.replace(".", "").replace("-", "").replace("rc", "").replace("ec", "").replace("fc", "")] = value
```

Comment:
Can you do this either via a regex or by a list of replacements?  This kind of code is hard to read.

Reply:
Fixed